src/templates/nav-primary.html

```diff
@@ -0,0 +1,31 @@
+<button class="usa-nav-close">
```

Comment:
This sounds like it fills a similar space as `usa-menu-btn`? Or maybe not at all, but unsure where this fits in.